arc/src/arc_curie_pme.c

```diff
@@ -187,12 +190,13 @@ void curie_pme_end_restore_mode(void)
 }
 
 // mark -- getter and setters--
-PATTERN_MATCHING_DISTANCE_MODE curie_pme_get_distance_mode(void) // L1 or LSup
+PATTERN_MATCHING_DISTANCE_MODE curie_pme_get_distance_mode(void)  // L1 or LSup
```

Comment:
I like the two spaces, to my eyes it makes it look more like a comment and less like an operation.

otoh i'm not sure what this comment means here and it could go *above* the function instead